moma_description/urdf/superpanda.urdf.xacro

```diff
@@ -81,7 +81,9 @@
   <xacro:arg name="control_mode" default='Position'/>
 
   <xacro:include filename="$(find moma_description)/urdf/panda.xacro"/>
-  <xacro:panda connected_to="arm_mount_link" use_fixed_realsense="$(arg use_fixed_realsense)" control_mode="$(arg control_mode)" tool="$(arg tool)" include_gazebo_control="true" robot_namespace="" control_namespace="control"/>
+  <xacro:panda connected_to="arm_mount_link" use_fixed_realsense="$(arg use_fixed_realsense)" 
```

Comment:
@grizzi Somehow the changes associated with these commits that touch the urdfs break the calibration.
I incorporated the new tool_frame location into my object grasping offset, but still, the keypoints that are detected do not match with the real position anymore by around 5 cm horizontally. Just be aware, didn't have time yet to figure out where this comes from.